mail_message_search/models/mail_thread.py

```diff
@@ -26,8 +26,10 @@ def _search_message_search(self, operator, value):
                 word_domain_list.append(expression.OR(field_domain_list))
         word_domain = expression.AND(word_domain_list)
         domain = expression.AND([[("model", "=", self._name)], word_domain])
-        messages = self.env["mail.message"].search(domain)
-        return [("id", "in", messages.mapped("res_id"))]
+        limit_value = self.env['ir.config_parameter'].sudo().get_param("mail.message.search.limit")
```

Comment:
Something like this may follow the convention better.
```suggestion
        limit_value = self.env['ir.config_parameter'].sudo().get_param("mail_message_search.message_limit")
```

Can you please add the CONFIGURE file for this?